layout/_macro/post.swig

```diff
@@ -166,6 +166,22 @@
                   <span class="post-comments-count join-count" itemprop="commentCount"></span>
                 </a>
               </span>
+
```

Comment:
very good.
I'm sorry I'm late.
Thank you for your improvement.
This is my first pull request,I'm so glad.